packages/dendron-cli/src/commands/notes.ts

```diff
@@ -32,28 +36,110 @@ type CommandOpts = CommandCLIOpts & SetupEngineResp & CommandCommonProps;
 
 type CommandOutput = { data: any; error?: DendronError };
 
+export type NoteCommandData = {
+  /**
+   * String output
+   */
+  stringOutput: string;
+  notesOutput: NoteProps[];
+};
+
 export enum NoteCommands {
+  /**
+   * Like lookup, but only look for notes
+   * Returns a list of notes
+   */
+  FIND = "find",
```

Comment:
I'm not a fan of introducing the terminology of 'find'.  It's inconsistent with 'lookup'.  This is unintuitive for users coming from the plugin.  I'd rather have `lookup` take additional parameters to specify note vs schema lookup.

Reply:
If we don't mind a breaking change, one option would be to name this lookup, and name the old lookup `goto` or something similar. That's more similar to what the old lookup CLI command does.

Reply:
i've switched this to `lookup` and renamed old lookup to `lookup_legacy`. we actually want to deprecate/remove `getNoteByPath` and add seperate `note get` and `note create` methods. there's less than 7 people currently using this method. will update docs for deprecation notice